frontend/src/pages/__snapshots__/GettingStarted.test.tsx.snap

```diff
@@ -5,7 +5,7 @@ exports[`GettingStarted page fallbacks to show pipeline list page if request fai
 - First value
 + Second value
 
-@@ -69,11 +69,13 @@
+@@ -70,11 +70,13 @@
```

Comment:
Looks like this is not a good way to do snapshot testing. I will consider alternatives.